src/nxdoc/nuxeo-server/administration/configuration-parameters-index-nuxeoconf.md

```diff
@@ -2364,6 +2364,18 @@ Set the maximum number of pages in the Nuxeo select element for pagination. If t
 
 * * *
 
+#### `org.nuxeo.web.ui.trustedDomains`
+
+Allows to specify a list of domains to which navigation is allowed when using a multi repository configuration. Takes a list of domains separated by a comma (e.g., hyland.com,hyland.net). Allows a redirect to any domain when left empty. Meant as a potential security enforcement option.
```

Comment:
Can we write like this?
*Enables users to specify a list of domains to which navigation is allowed when using a multi-repository configuration. Takes a list of domains separated by a comma (for example, hyland.com,hyland.net). Allows a redirect to any domain when left empty. Meant as a potential security enforcement option.*

Reply:
updated content